src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/ApprovalStep.tsx

```diff
@@ -147,9 +154,13 @@ const ApprovalStepPending = ({
   const handleSignAllowanceApproval = useCallback(async () => {
     // Only proceed to execute the approval if the promise is resolved, i.e the user has opened the
     // Ledger app without cancelling
-    await checkLedgerAppOpenIfLedgerConnected(tradeQuoteStep.sellAsset.chainId)
-      .then(() => approveMutation.mutateAsync())
-      .catch(console.error)
+    try {
+      await checkLedgerAppOpenIfLedgerConnected(tradeQuoteStep.sellAsset.chainId)
+      setFeeQueryEnabled(false)
+      await approveMutation.mutateAsync()
+    } catch (error) {
+      console.error(error)
```

Comment:
It would be wise to prevent against this for sure - I committed your suggestion in https://github.com/shapeshift/web/pull/7539/commits/9daf5eb9462bc2f2e189b02b5571fa067b2e62b5 🙏